dashboard/src/pages/EventCfp.vue

```diff
@@ -28,66 +28,70 @@ const event = createDocumentResource({
   auto: true,
 })
 
-let tabs = reactive({
+const tabs = reactive({
   options: [
     {
       label: 'Overview',
       route: `/event/${route.params.id}/cfp`,
     },
-    {
-      label: 'Web Form',
-      route: `/event/${route.params.id}/cfp/create`,
-    },
   ],
 })
 
-const replaceCreateOption = () => {
-  tabs.options = tabs.options.filter((d) => d.label !== 'Web Form')
-
-    if (!tabs.options.find((d) => d.label === 'Web Form')){
-        tabs.options.push({
-            label: 'Web Form',
-            route: `/event/${route.params.id}/cfp/edit`
-        })
-    }
-
-    if (!tabs.options.find((d) => d.label === 'Insights')){
-        tabs.options.push({
-            label: 'Insights',
-            route: `/event/${route.params.id}/cfp/insights`
-        })
-    }
-}
-
-let cfp_exists = ref(false)
-let event_cfp = reactive({})
-
-watch(event, (newEvent) => {
-  event_cfp = createResource({
-    url: 'frappe.client.get',
-    params: {
+const has_cfp = createResource({
+  url: 'frappe.client.get_count',
+  makeParams() {
+    return {
       doctype: 'FOSS Event CFP',
       filters: {
-        event: newEvent.doc.name,
+        event: route.params.id,
       },
+    }
+  },
+  auto: true,
+  onSuccess(data) {
+    resetTabs()
+    if (data > 0) {
+      event_cfp.fetch()
+      tabs.options.push(...[
```

Comment:
```suggestion
      tabs.options.push(...[
```

Why the spread?

Reply:
without the spread it only shows the `Overview` tab